androidapp/app/src/main/java/com/example/lanweather/data/entity/PeriodEntity.kt

```diff
@@ -0,0 +1,55 @@
+package com.example.lanweather.data.entity
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.Ignore
+import androidx.room.PrimaryKey
+import com.example.lanweather.data.model.Period
+
+@Entity(tableName = "Periods")
+data class PeriodEntity(
```

Comment:
Just to clarify, does the PeriodEntity contains the weather data for a specific hour?

Reply:
it is used by both the Hourly and Daily objects. So the interval isn't a constant length of time.

Reply:
Period Entity is weather data over the period specified by the member fields StartTime and EndTime